doc/lsst.fgcmcal/tasks/lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask.rst

```diff
@@ -0,0 +1,125 @@
+.. lsst-task-topic:: lsst.fgcmcal.fgcmBuildStars.FgcmBuildStarsTask
+
+##################
+FgcmBuildStarsTask
+##################
+
+``FgcmBuildStarsTask`` finds all the single-visit sources in a repository (or a subset based on command-line parameters) and extracts all the potential photometric calibration stars for input into fgcm.  This task additionally uses fgcm to match star observations into unique stars, and performs as much cleaning of the input catalog as possible.  A tutorial on the steps of running ``fgcmcal`` are found in the `cookbook <https://github.com/lsst/fgcmcal/tree/master/cookbook>`_.
+
+The ``fgcmcal`` code runs on calexp source catalogs from visits constrained by the ``--id`` parameter on the command line.  Best results are obtained when ``fgcmcal`` is run with full visits.
+
+In Gen2, due to limitations of the Gen2 Butler, optimal performance is obtained by specifying a single "reference" ccd on the command line (e.g. ``ccd=13``) and setting the config variable ``checkAllCcds = True`` (which is the default).  The alternative is to specify all the desired CCDs and set ``checkAllCcds = False``, e.g., ``ccd=0..8^10..103``.  However, this is slower than the first option, and the improvement in speed in the first option is greater the more visits are specified.  If instead you want to process all the visits in a rerun selected by filter, field, or some other dataid field, then by using a reference ccd and setting ``checkAllCcds = True`` you can speed things up by a factor of approximately 100 relative to the alternative (naming CCDs specifically).
```

Comment:
About the Gen2 trick of specifying a single ccd, does it lead to missing the visits of which the specified CCD are missing despite calexp/src of other CCDs exist? If so, it might be worth noting. 